### ISSUE TITLE
Metadata not saving for new entries

### ISSUE THREAD

**Issue by BenParizek** [barrelstrength/craft-sprout-seo#192]
Update behavior of normalizeValue

<img src="https://frontapp.com/assets/img/favicons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_1cshv)

<img src="https://frontapp.com/assets/img/favicons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_1d2wz)

**Comment by mattstein** [comment]
I’m looking at a project on Craft 3.4.8 + Sprout SEO 4.3.3 where the `content` table’s `fields_basicSeo` column has a blob of `null` and empty (`""`) values except for the `optimizedTitle`. It also looks like older posts have more complete/intact JSON and this applies to new or newly-saved Entries.

Same thing?

**Comment by BenParizek** [comment]
@mattstein Yes. We should have a fix out soon.

**Comment by rsoule** [comment]
@BenParizek Any update on this? We're currently blocked by this and can't make any updates across the site without all our current Meta data being erased. Any update would be much appreciated.

**Comment by BenParizek** [comment]
@rsoule Currently testing the fix. Hoping to have a release out later today.

**Comment by BenParizek** [comment]
@rsoule We've just released Sprout SEO 4.4.1 which addresses a few issues about the way metadata is was being saved in the latest release adding Craft 3.4 support.

This release included several updates to how the underlying metadata is managed. Feel free to open another ticket if you continue to run into anything after the upgrade.
